src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -97,6 +97,12 @@ public void MakeRelativeTests()
                 Assert.Equal(@"\\host\path\file", FileUtilities.MakeRelative(@"c:\abc\def", @"\\host\path\file"));
                 Assert.Equal(@"\\host\d$\file", FileUtilities.MakeRelative(@"c:\abc\def", @"\\host\d$\file"));
                 Assert.Equal(@"..\fff\ggg.hh", FileUtilities.MakeRelative(@"c:\foo\bar\..\abc\cde", @"c:\foo\bar\..\abc\fff\ggg.hh"));
+
+                /* Directories */
+                Assert.Equal(@"def\", FileUtilities.MakeRelative(@"c:\abc\", @"c:\abc\def\"));
+                Assert.Equal(@"..\", FileUtilities.MakeRelative(@"c:\abc\def\xyz\", @"c:\abc\def\"));
+                Assert.Equal(@"..\ttt\", FileUtilities.MakeRelative(@"c:\abc\def\xyz\", @"c:\abc\def\ttt\"));
+                Assert.Equal(@".", FileUtilities.MakeRelative(@"c:\abc\def\", @"c:\abc\def\"));
```

Comment:
There is opportunity to extend test coverage for combination of file+directory and directory+file.
```
/* Directory + File */
 Assert.Equal(@"def", FileUtilities.MakeRelative(@"c:\abc\", @"c:\abc\def"));
 Assert.Equal(@"..\..\def", FileUtilities.MakeRelative(@c:\abc\def\xyz\", @"c:\abc\def"));
 Assert.Equal(@"..\ghi", FileUtilities.MakeRelative(@c:\abc\def\xyz\", @"c:\abc\def\ghi"));
 Assert.Equal(@"..\def", FileUtilities.MakeRelative(@c:\abc\def\", @"c:\abc\def"));

/* File + Directory */
 Assert.Equal(@"def\", FileUtilities.MakeRelative(@c:\abc", @"c:\abc\def\"));
 Assert.Equal(@"..\", FileUtilities.MakeRelative(@c:\abc\def\xyz", @"c:\abc\def\"));
 Assert.Equal(@"..\ghi\", FileUtilities.MakeRelative(@c:\abc\def\xyz", @"c:\abc\def\ghi\"));
 Assert.Equal(@"", FileUtilities.MakeRelative(@c:\abc\def", @"c:\abc\def\"));
```

Reply:
Good suggestion - have added more tests